var/spack/repos/builtin/packages/py-geopandas/package.py

```diff
@@ -42,21 +44,27 @@ class PyGeopandas(PythonPackage):
     depends_on("python@3.9:", type=("build", "run"), when="@0.14:")
     depends_on("py-setuptools", type="build")
     depends_on("py-setuptools@61.0.0:", type="build", when="@0.14:")
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-numpy@1.22:", type=("build", "run"), when="@0.14.4:")
+    # Only for versions 0.x.y - replaced by py-pyogrio
+    depends_on("py-fiona", type=("build", "run"), when="@:0.99")
```

Comment:
(no change needed) you can drop the `9`'s if desired like 

```
depends_on("py-fiona", type=("build", "run"), when="@:0")
```

Reply:
Good point. Will remember next time! I always do this for @:1 i.e. anything >0 - for some reason it didn't occur to me that this works for @:0 as well.